broadcast/src/linked/serializer.rs

```diff
@@ -0,0 +1,26 @@
+use super::{safe, Epoch};
+use bytes::BufMut;
+use commonware_cryptography::Array;
+
+/// Serializes an Ack message into a byte array.
+pub fn ack<D: Array, P: Array>(chunk: &safe::Chunk<D, P>, epoch: Epoch) -> Vec<u8> {
+    let len = chunk.sequencer.len() + size_of::<u64>() + chunk.payload.len() + size_of::<u64>();
```

Comment:
You should avoid using `size_of` and use the `SERIALIZED_LEN` both for `u64` and the sequencer/payload fields.